packages/amplify-category-notifications/lib/channel-APNS.js

```diff
@@ -156,11 +156,11 @@ function pull(context, pinpointApp) {
     ApplicationId: pinpointApp.Id,
   };
   spinner.start(`Pulling ${channelName} Channel.`);
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve, reject) => {//eslint-disable-line
     context.exeInfo.pinpointClient.getApnsChannel(params, (err, data) => {
       if (err) {
-        spinner.fail(`get channel ${channelName} error`);
-        reject(err);
+        spinner.succeed(`no channel data retrieved for: ${channelName}`);
```

Comment:
Why is this a success instead of err? Why are we resolving this?

Reply:
In the Pinpoint API, if a channel has not been properly setup (has never been enabled), the get channel method errs. For our purpose, we just don't set the channel in the pinpointApp object and resolve it.

Reply:
This part has been changed so be more precise on that err to ignore.